resources/sdk/pureclouddotnet/templates/Configuration.mustache

```diff
@@ -121,6 +121,7 @@ namespace {{packageName}}.Client
 
                 ApiClient = apiClient;
             }
+            ApiClient.RetryConfig = new ApiClient.RetryConfiguration();
```

Comment:
This will overwrite whatever RetryConfig was initially set on the apiClient passed in to this method. Instead of setting it here, add this to each of the ApiClient class's constructors. Or even better, don't set a default config since it doesn't do anything and make sure the code handles the property being null (i.e. gracefully doesn't retry).

Reply:
fixed with constructor initialization.